pinot-common/src/main/java/org/apache/pinot/common/function/FunctionUtils.java

```diff
@@ -42,6 +42,11 @@ private FunctionUtils() {
     put(Double.class, PinotDataType.DOUBLE);
     put(String.class, PinotDataType.STRING);
     put(byte[].class, PinotDataType.BYTES);
+    put(int[].class, PinotDataType.INTEGER_ARRAY);
```

Comment:
Can we keep the same order as the SV ones (int, long, float, double, string), same for other places

Reply:
I have made the order same as SV(int, long, float, double, string)